ci/deploy-to-production

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Echo all commands just before they are executed
+set -x
+
+# Stop processing this script if any of the commands we run return an error
+# (i.e. exit with a non-zero status)
+set -e
+
+cd /var/www
+
+# Creates a deployment request for Production
+# The deployment gets queued and will be waiting for approval
+# from a release manager in CWP's dashboard,
+# as per the `bypass_and_start` parameter in the request.
+curl -X POST -u $CWP_DASH_USER:$CWP_DASH_TOKEN \
+  https://dash.cwp.govt.nz/naut/project/$CWP_STACK_ID/environment/prod/deploys \
+  -H 'Content-Type: application/json' -H 'Accept: application/json' \
+  --data "{ \
```

Comment:
🐘 this would be good to have in its own JSON file, which we could use `envsubt` to replace the variables (or `sed`).

Reply:
The CWP specific deployment steps need to be fixed in bitbucket first through WCON cause otherwise I won't be able to code and test.